controllers/external-accounts.js

```diff
@@ -41,4 +44,49 @@ const getExternalAccountData = async (req, res) => {
   }
 };
 
-module.exports = { addExternalAccountData, getExternalAccountData };
+/**
+ * Gets all group-roles
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const syncExternalAccountData = async (req, res) => {
+  try {
+    const [discordUserData, rdsUserData] = await Promise.all([getDiscordMembers(), getDiscordUsers()]);
+    const rdsUserDataMap = {};
+    const userDataUpdatorPromises = [];
```

Comment:
I am not sure, both convey the same meaning.

Reply:
- Is the word `updator` the right one to use here? Is it a word at first?

Reply:
oops its a typo, i can get it fixed it should be `updater`

Reply:
Are there any changes apart from that that you would like to mention? So that they can go in a single commit rather than having multiple commits. @yesyash